slackapi/deno-slack-sdk
# Improve integration tests

### Summary

This PR introduces an integration test that ensure that the output files of the generation script are always valid typescript. The goal of this test is to ensure no breaking changes to the generation script are introduced inadvertently.

The integration tests are not included in the code coverage but will get executed by the CI pipeline.

### testing

1. pull this branch
2. run deno task test to execute all tests
3. Ensure `tests/integration/schema/slack/functions/_scripts/write_function_files_test.ts` gets executed
4. run deno task coverage to execute the code coverage
5. Follow the steps in `src/schema/slack/functions/_scripts/README.md` to try out the generation script and ensure everything still works

### Requirements 

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/deno-slack-sdk/blob/main/.github/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've ran `deno task test` after making the changes.
